Backend/app/dashboard/Lib/src/Domain/Action/Dashboard/Person.hs

```diff
@@ -246,6 +264,46 @@ getAccessMatrix tokenInfo = do
   accessMatrixItems <- runInReplica $ QMatrix.findAllByRoleId encPerson.roleId
   pure $ DMatrix.mkAccessMatrixRowAPIEntity accessMatrixItems role
 
+changePasswordByAdmin ::
+  (EsqDBFlow m r, EncFlow m r) =>
+  TokenInfo ->
+  Id DP.Person ->
+  ChangePasswordByAdminReq ->
+  m APISuccess
+changePasswordByAdmin _ personId req = do
+  unless (req.newPassword == req.reEnterPassword) . throwError $ InvalidRequest "Password do NOT match"
```

Comment:
Shouldn't be  this check on frontend side?

Reply:
@roman-bodavskiy , yeah frontend will check , just put this condition from backend side also so that , if while testing from postman or hitting API directly , in that case backend should also check

Reply:
Not required at backend.